ansible_collections/arista/avd/roles/upgrade_tools/templates/upgrade-2.x-to-3.0.j2

```diff
@@ -404,3 +404,24 @@ evpn_rt_type:
 evpn_rt_type:
   admin_subfield: "bgp_as"
 {% endif %}
+{##}
```

Comment:
Expand to comment or change for newline

Reply:
This was just to avoid getting an empty line in all the upgrade files for every device, but yes I will change it.